container/secrets.go

```diff
@@ -81,5 +86,8 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	if _, err := os.Stat(overrideDir); err == nil {
```

Comment:
@rhatdan @cgwalters case:
```
/usr/share/rhel/secrets/rhsm/ <- not empty
/etc/container/rhsm/ <- EMPTY
```
what's the end result in the container?
an empty dir or the merge? I'd say the empty dir, the other way is too much maybe or not?